packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -3302,5 +3302,34 @@ describe("calcite-combobox", () => {
       };
       themed(singleSelectComboboxHTML, comboboxTokens);
     });
+
+    const comboboxSelectAllEnabledHTML = html`
+      <calcite-combobox select-all-enabled>
+          </calcite-combobox-item value="Pine" text-label="Pine">
+          </calcite-combobox-item value="Not Pine" text-label="Not Pine">
+        </calcite-combobox>
+    `;
+
+    describe("select-all-enabled", () => {
+      const comboboxTokens: ComponentTestTokens = {
+        "--calcite-combobox-divider-color": {
+          shadowSelector: `.${CSS.selectAll}`,
+          selector: "calcite-combobox",
+          targetProp: "borderBlockEndColor",
+        },
+      };
+      themed(comboboxSelectAllEnabledHTML, comboboxTokens);
+    });
+
+    describe("deprecated", () => {
+      const comboboxTokens: ComponentTestTokens = {
```

Comment:
Suggestion: You can simplify `themed` tests by inlining the test tokens object (see [example](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/components/icon/icon.e2e.ts#L126-L130)). Totally up to you.